src/views/Viewer.vue

```diff
@@ -850,7 +850,7 @@ export default {
 }
 
 // force white icon on single buttons
-.action-item--single.icon-menu-sidebar {
+#viewer .action-item--single.icon-menu-sidebar {
```

Comment:
To be honest, this is fixed with the new Icons fix of the vue components v4 as we're now using the css colour. :thinking: 
Do you want to wait for the release?

Reply:
Ah, well, that still applies for 21 and under